src/subscription_manager/release.py

```diff
@@ -81,22 +81,28 @@ def __init__(self):
     def get_releases(self):
         # cdn base url
 
-        # find the rhel product
-        release_product = None
+        # Find the rhel products
+        release_products = []
         installed_products = self.product_dir.get_installed_products()
         for product_hash in installed_products:
             product_cert = installed_products[product_hash]
             products = product_cert.products
             for product in products:
                 rhel_matcher = rhelproduct.RHELProductMatcher(product)
                 if rhel_matcher.is_rhel():
-                    release_product = product
+                    release_products.append(product)
 
-        if release_product is None:
-            log.debug("No products with RHEL product tags found")
+        if len(release_products) == 0:
+            log.info("No products with RHEL product tags found")
             return []
+        elif len(release_products) > 1:
+            log.error("More then one product with RHEL product tags found")
+            return None
```

Comment:
Not 100% sure if this is the best approach (returning [] and None for "error" states), but I wasn't able to invent better approach. Other `release_provider` can be `ApiReleaseVersionProvider` and I think it doesn't return None at all.